internal/serve/httphandler/profile_handler.go

```diff
@@ -49,6 +49,7 @@ type PatchOrganizationProfileRequest struct {
 	TimezoneUTCOffset              string  `json:"timezone_utc_offset"`
 	IsApprovalRequired             *bool   `json:"is_approval_required"`
 	SMSResendInterval              *int64  `json:"sms_resend_interval"`
+	PaymentCancellationPeriod      *int64  `json:"payment_cancellation_period_days"`
```

Comment:
You've changed the json tag name but not the struct attribute.

```suggestion
	PaymentCancellationPeriodDays      *int64  `json:"payment_cancellation_period_days"`
```